### PR TITLE
my side of the story

### PR DESCRIPTION
**Pull request by AsharKamal98**

### CODE REVIEW

**Comment by floridop** [comment]
Hi 

I do not accept changes that do not include any contribution. I need you to contribute with something. Please integrate your contribution with the current main status. You may do one of these two things, just to experiment:
- rollback to the latest commit using the techniques described in Tutorial 5 slides 108-110 on your mystory branch, and _force push_ to that branch on github,

**Comment by AsharKamal98** [comment]
I did the first option, but accidentally used an older commit hash when doing so. So I had to manually put back in your original line and then do the push.

File: fairytale.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ As she was doing so, a sudden ray of lighting reflected over the
 Fox Interstellar News logo on the device, blasting instant light on 
 Zo-oZ's terrified expression. 
 
-"I don't want to believe what we've just seen" - said Zo-oZ - "I hope I will 
+"I can believe what we've just seen" - said Zo-oZ - "I hope I will
```
Comment (floridop) [inline fairytale.md:12]: Please change this back to "don't want to"

File: fairytale.md [hunk 2]
```diff
@@ -18,8 +18,7 @@ stuff. I thought you said you were used to it!"
 "Did I say...? Oh sure, sure! I've seen this before, no problem!" - 
 answered Zo-oZ in a mix of shame and confusion.
 
-As they stepped into the company's instaporter, the lights of the
-surrounding lamps started blurring and fading...
+As they stepped into the company's instaporter, the lights of the surrounding lamps started blurring and fading. A strange feeling came over Zo-oZ.
```
Comment (floridop) [inline fairytale.md:21]: This addition is good, please keep

**Review by floridop — CHANGES_REQUESTED** [review]
Hi @AsharKamal98 

The current state looks good, but the result is still not mergeable. When you added manually my content you generated a new conflict. Please follow the requested steps in the review comments.

You still have to perform the merge on your side.
Could you please perform a merge as described in Tutorial 5 slides 82-96 and integrate those changes?
Start with synchronizing your local main with my upstream main as described in slide 86.
We do not want to lose any existing content, so you should keep what others have written and add yours.
You're free to reorganize the content blocks so that it still makes a story, but please do minimal changes
to the content of what others have written to integrate them with yours. 
You have time until the 25th of October at 12:00. 
If you can't make it by that time or you are done before the deadline, 
please add a comment in github so that I can progress with another student.

**Comment by floridop** [comment]
I noticed that you did not sync your fork origin main to my upstream main. It means you're not following the slides. It is up to you to decide how much to make of this git course, it seems to me you had previous knowledge of git that you're reusing, but I believe you do not have a clear understanding of the git-flow we're following.

**Comment by AsharKamal98** [comment]
This is my first time using github, I made the last change in a hurry since I was running late on time. I'll try doing it more carefully following the slides this time.

**Comment by floridop** [comment]
> This is my first time using github, I made the last change in a hurry since I was running late on time. I'll try doing it more carefully following the slides this time.

Please let me know if you need more time. The purpose of the exercise is to learn, no need to rush. But I cannot keep your colleagues waiting for too long, as the end of the course is approaching. If you want I can go ahead with others and come back to you in a later time.

**Comment by AsharKamal98** [comment]
> 

Thanks, sure you can go ahead with the others and take me later. This will give me time to sync the repositories as well. Sorry for the delay!

**Comment by floridop** [comment]
thanks for the quick answer! I will update on canvas so I remember.

**Comment by floridop** [comment]
Hi! Thanks for your submission.

Unfortunately it seems that there were changes in the main that conflict with yours. 
Could you please perform a merge as described in Tutorial 5 slides 82-96 and integrate those changes?
Start with synchronizing your local main with my upstream main as described in slide 86.
We do not want to lose any existing content, so you should keep what others have written and add yours.
You're free to reorganize the content blocks so that it still makes a story, but please do minimal changes
to the content of what others have written to integrate them with yours. 
The merges will be approved according to arrival time starting from October 26th, 16.00, 
so expect that you need to merge again after that time.
If I do not approve the changes you will be asked to update the content.

**Comment by floridop** [comment]
This PR is closed as the student never gave input.
Please ask if you need it reopened.

**Pull request closed without merging**